configs/gca/gca_dimaug-r34_200k-4xb10_comp1k.py

```diff
@@ -1,4 +1,8 @@
-_base_ = ['./gca_r34_4x10_200k_comp1k.py']
+_base_ = ['./gca_r34_200k-4xb10_comp1k.py']
+
+experiment_name = 'gca_dimaug-r34_200k-4xb10_comp1k'
```

Comment:
`dimaug` is related to augmentation. Maybe place this in `training setting`?

Reply:
ok